### PR TITLE
predator-prey-size-marine

### PR DESCRIPTION
**Pull request by henrykironde**

### CODE REVIEW

File: scripts/predator_prey_size_marine.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+from retriever.lib.templates import Script
+
+
+class main(Script):
+    def __init__(self, **kwargs):
+        Script.__init__(self, **kwargs)
+        self.name = "Marine Predator and Prey Body Sizes - Barnes et al. 2008"
+        self.shortname = "predator-prey-size-marine"
+        self.retriever_minimum_version = '2.0.dev'
+        self.version = '1.2.0'
```
Comment (ethanwhite) [inline scripts/predator_prey_size_marine.py]: I think we should go to `2.0` here since it's a completely new script.

**Comment by ethanwhite** [comment]
Oops, didn't see that you'd bumped the version. In it goes.

**Comment by henrykironde** [comment]
wait

**Comment by ethanwhite** [comment]
Oh wait, are we waiting on #897 before merging this?

**Comment by henrykironde** [comment]
yes

**Comment by ethanwhite** [comment]
Got it, sorry, forgot.

**Comment by jainamritanshu** [comment]
I am sorry for the delay in #897. I am working on it and will make the changes asap,

**Comment by henrykironde** [comment]
@ethanwhite updated

**Pull request merged**